example_10/doc/userdoc.rst

```diff
@@ -154,22 +217,34 @@ The *RRBot* URDF files can be found in the ``description/urdf`` folder.
 
   .. code-block:: shell
 
-    ros2 topic echo /gpio_controller/inputs
+    ros2 topic echo /gpio_controller/gpio_states
 
   again and you should see that - unless commands are received - the values of the state interfaces are now ``nan`` except for the vacuum interface.
 
   .. code-block:: shell
 
-    interface_names:
-    - flange_analog_IOs/analog_output1
-    - flange_analog_IOs/analog_input1
-    - flange_analog_IOs/analog_input2
-    - flange_vacuum/vacuum
-    values:
-    - .nan
-    - .nan
-    - .nan
-    - 1.0
+    header:
+      stamp:
+        sec: 1731875298
+        nanosec: 783713170
+      frame_id: ''
+    interface_groups:
+    - flange_analog_IOs
+    - flange_vacuum
+    interface_values:
+    - interface_names:
+      - analog_output1
+      - analog_input1
+      - analog_input2
+      values:
+      - .nan
+      - .nan
+      - .nan
+    - interface_names:
+      - vacuum
+      values:
+      - 1.0
+    ---
```

Comment:
When I've checked this scenario I got following output on this step. I believe we already updated gpios' states. 
```
header:
  stamp:
    sec: 1732039620
    nanosec: 696716529
  frame_id: ''
interface_groups:
- flange_analog_IOs
- flange_vacuum
interface_values:
- interface_names:
  - analog_output1
  - analog_input1
  - analog_input2
  values:
  - 0.5
  - 1286567808.0
  - 1762263040.0
- interface_names:
  - vacuum
  values:
  - 0.0
  ```

Reply:
only if you have the publisher running while running the new launchfile?